### PR TITLE
Update mvc.component.html

### PR DESCRIPTION
**Pull request by Abrissirba**
I had to install fastify-formbody and fastify-static aswell

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

## Other information

### CODE REVIEW

File: src/app/homepage/pages/techniques/mvc/mvc.component.html [hunk 1]
```diff
@@ -62,7 +62,7 @@ <h4>Fastify</h4>
     library. In order to create a MVC application with fastify, we have to install following packages:
   </p>
   <pre><code class="language-bash">
-$ npm i --save fastify point-of-view handlebars</code></pre>
+$ npm i --save fastify fastify-formbody fastify-static point-of-view handlebars</code></pre>
```
Comment (johannesschobel) [inline src/app/homepage/pages/techniques/mvc/mvc.component.html]: Is this change really necessary? I mean, those packages are optional and may depend on the specific scenario of the application. E.g., if i am building a REST API only, i may not need the `fastify-formbody` or `fastify-static` packages.

I think, it would be better to just highlight those packages (or rather the overall `fastify` website) in a specific `hint` block?

**Comment by kamilmysliwiec** [comment]
As @johannesschobel pointed out, these packages are optional. A user will see appropriate notifications in the console if they are needed.
